src/FlashMessage/FlashMessage.tsx

```diff
@@ -42,55 +42,43 @@ const FlashMessage = ({ flashMessage }: FlashMessageProps) => {
     }
 
     const close = () => {
-        if (timeoutHandler.current) {
-            clearTimeout(timeoutHandler.current);
-        }
+        clearTimeout(timeoutHandler.current);
         dispatch(removeMessage(id));
     };
 
     const addFadeout = () => {
-        if (dismissTime != null) {
-            timeoutHandler.current = setTimeout(() => {
-                dispatch(removeMessage(id));
-            }, dismissTime);
+        if (dismissTime) {
+            timeoutHandler.current = setTimeout(
+                () => dispatch(removeMessage(id)),
+                dismissTime
+            );
+            setFadeoutTimer(`${dismissTime}ms`);
         }
-        setFadeoutTimer(`${dismissTime}ms`);
     };
 
     const removeFadeout = () => {
         setFadeoutTimer('unset');
-        if (timeoutHandler.current) {
-            clearTimeout(timeoutHandler.current);
-        }
+        clearTimeout(timeoutHandler.current);
     };
 
-    const initialRender = () => divRef.current == null;
-
     return (
         <div
             ref={divRef}
             style={{
                 backgroundColor: getBackgroundColorFromVariant(variant),
                 color: colors.white,
                 zIndex: 1000,
-                animation: initialRender() ? 'slide-in 1s' : 'unset',
+                animation: !divRef.current ? 'slide-in 1s' : 'unset',
                 width: '100%',
                 padding: '16px',
                 display: 'flex',
                 flexDirection: 'column',
                 justifyContent: 'space-between',
             }}
-            className={`flash-variant-${variant}`}
             onMouseEnter={removeFadeout}
             onMouseLeave={addFadeout}
         >
-            <div
-                style={{
-                    width: '100%',
-                    display: 'flex',
-                    justifyContent: 'space-between',
-                }}
-            >
+            <div className="d-flex w-100 justify-space-between">
```

Comment:
I thought we just agreed to not add more bootstrap utility classes, as we will port to tailwind?